### ISSUE TITLE
RGBHSW params - Not clear to me

### ISSUE THREAD

**Issue by vampywiz17** [make-all/tuya-local#130]
Hello there,

I read the new light setting in dokumentation. But one thing not clear to me:

"rgbhsv (optional, hex): a dps to control the color of the light, using 14 digit hex encoding of RGB and HSV values."

My HSV set is :

` "values": "{\"h\":{\"min\":1,\"scale\":0,\"unit\":\"\",\"max\":360,\"step\":1},\"s\":{\"min\":1,\"scale\":0,\"unit\":\"\",\"max\":255,\"step\":1},\"v\":{\"min\":1,\"scale\":0,\"unit\":\"\",\"max\":255,\"step\":1}}"`

But it three different value. How to set the min/max in this case?

**Comment by make-all** [comment]
It needs some tweaking, as the first implementation had max of 100 for s and v, and that got hardcoded.  Probably I will add an additional config option for "format" of these hex encoded entries, something like:
```
format:
  - name: red
    bytes: 1
    range:
       min: 0
       max: 255
  - name: green
    bytes: 1
    range:
       min: 0
       max: 255
  - name: blue
    bytes: 1
    range:
       min: 0
       max: 255
   - name: hue
     bytes: 2
     range:
        min: 0
        max: 360
   - name: saturation
     bytes: 1
     range:
       min: 0
       max: 255
    - name: value
      bytes: 1
      range:
        min: 0
        max: 255
```

**Comment by make-all** [comment]
Support for the format specification detailed above is released in 0.16.0, with the Digoo and MoesHome smartplugs with nightlight both using this with different ranges for saturation and value.  For names, I just used the single character names (r, g, b, h, s, v) instead of spelling them out, and if the range is 0-255 (for 1 byte subfields, 65535 for 2 byte etc) the range can be omitted.
